src/Tools/IdeBenchmarks/IdeBenchmarks.csproj

```diff
@@ -11,6 +11,7 @@
     <NoWarn>$(NoWarn),CA2007</NoWarn>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="PresentationFramework" />
```

Comment:
what is this?  needed?

Reply:
yeah.  this reference is now no longer pulled in automatically through the transitive reference, and this project actually uses it.  so had to add an explicit reference.  @jasonmalinowski has more context.

Reply:
@CyrusNajmabadi The better fix actually is to put `<UseWpf>true</UseWpf>` in your project file like we do in other projects.

Reply:
will do.